ajax/libs/jshashtable.js/3.0/hashtable.js

```diff
@@ -0,0 +1,402 @@
+/**
```

Comment:
This file's md5 hash is `041054fb273de9c0ecdbaeef262050d9`
But the origin hash is `b2d65ae2edd7a7f7c751b2f02f76fb1e`
Would you please re-add this file and generate the minified files again?
Thank you.